admin/deployments.go

```diff
@@ -177,6 +177,54 @@ func (s *Service) updateDeployment(ctx context.Context, depl *database.Deploymen
 	return nil
 }
 
+// HIBERNATE free deployments
+func (s *Service) HibernateDeployments(ctx context.Context) error {
```

Comment:
nit: Update docstring to `// HibernateDeployments tears down unused deployments`